backend/src/appointment/routes/account.py

```diff
@@ -27,7 +27,7 @@ def get_external_connections(subscriber: Subscriber = Depends(get_subscriber)):
     external_connections = defaultdict(list)
 
     if os.getenv('ZOOM_API_ENABLED'):
-        external_connections['Zoom'] = []
+        external_connections['zoom'] = []
```

Comment:
I hope this won't cause any issues (it didn't in my local environment). Since fxa and google are lowercase, I thought I just change this for convenience.

Reply:
I did a case sensitive search for `Zoom` over the whole codebase, but couldn't find another place.